augurpy/estimators.py

```diff
@@ -1,25 +1,57 @@
-from typing import Dict, List, Optional, Union
+from typing import Dict, Literal, Union
 
 from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
 from sklearn.linear_model import LogisticRegression
+from typing_extensions import TypedDict
+
+
+class Params(TypedDict, total=False):
```

Comment:
Could this be a Dataclass?

Reply:
Yes! The decorator you mean, right? Done.

Reply:
Yes, the decorator. The params object looks like it's purely a data container but does not need any functionality.